pebbles/models.py

```diff
@@ -489,6 +489,20 @@ def generate_name(prefix):
         return '%s%s-the-%s' % (prefix, names.get_first_name().lower(), random.choice(NAME_ADJECTIVES))
 
 
+class InstanceLog(db.Model):
+    __tablename__ = 'instance_logs'
+    id = db.Column(db.String(32), primary_key=True)
+    instance_id = db.Column(db.String(32), db.ForeignKey('instances.id'), index=True, unique=False)
+    log_level = db.Column(db.String(8))
+    log_type = db.Column(db.String(64))
+    timestamp = db.Column(db.Float)
+    message = db.Column(db.String(256))
```

Comment:
After a minute of thought if the full messages are logged to a file then a somewhat longer string (1024) and an explicit clamping of message length would be a possible alternative.